src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -65,7 +65,7 @@
 
 	GrinderLogging( "Misc", false ), Logging( "Misc" ), IntegrationLogging( "Misc", false ), WebsiteRecipes( "Misc", false ),
 
-	enableFacadeCrafting( "Crafting" ), inWorldSingularity( "Crafting" ), inWorldFluix( "Crafting" ), inWorldPurification( "Crafting" ), UpdateLogging( "Misc", false ),
+	enableFacadeCrafting( "Crafting" ), enableWirelessTerminalColoring( "Crafting" ), inWorldSingularity( "Crafting" ), inWorldFluix( "Crafting" ), inWorldPurification( "Crafting" ), UpdateLogging( "Misc", false ),
```

Comment:
I'm assuming that this, as well as the other similar complaints, are alright (as they match the other constants)